test/conformance/chainsaw/globalcontext/apicall-failed/clusterpolicy.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: cpol-apicall-failed
+spec:
+  failurePolicy: Fail
+  rules:
+  - name: main-deployment-exists
+    context:
+    - name: deploymentCount
+      globalReference:
+        name: gctxentry-apicall-failed
+        jmesPath: "length(@)"
```

Comment:
This [GCE](https://github.com/kyverno/kyverno/blob/40825b30abb4f3aaa488f1320baa50c5c4975358/test/conformance/chainsaw/globalcontext/apicall-failed/gctxentry.yaml) looks up unknown namespaces, how is it related to deployment count?

Reply:
This is a failing test. The idea is to show an error not the actual query itself.